cmd/version.go

```diff
@@ -15,8 +21,45 @@ import (
 
 const (
 	versionTimeout = time.Second * 2
+	clientVersion  = time.Second * 1
+	apiURL         = "https://api.github.com/repos/odpf/optimus/releases/latest"
 )
 
+func checkLatestVersion() string {
+	gitClient := http.Client{
+		Timeout: clientVersion,
+	}
+
+	req, err := http.NewRequest(http.MethodGet, apiURL, nil)
+	if err != nil {
+		log.Warn("failed to create request for latest version")
```

Comment:
If an error occurs it should not continue to execute the function and return an empty string

Reply:
But i think we discussed, that the execution of the command should not stop regardless of the error , we will just warn the user about the error

Reply:
we are not stopping the execution of the command but how do you expect this function to work if it failed to create even a valid http request.